torch/_dynamo/codegen.py

```diff
@@ -345,3 +348,15 @@ def load_import_from(self, module_name, object_name):
 
     def create_begin_finally(self):
         return create_instruction("BEGIN_FINALLY")
+
+    def create_call_function_kw(self, nargs, kw_names, push_null):
+        if sys.version_info >= (3, 11):
+            return [
+                create_instruction(
+                    "KW_NAMES", self.get_const_index(self.code_options, kw_names)
```

Comment:
Shouldn't we set up all the stack, then call `KW_NAMES` -> `PRECALL` -> `CALL` without anything between them?
Is the push_null+rotn in the create_call_function below handling that properly?